### ISSUE TITLE
[application/x-diskcopy, 6] The file doesn't fit

### ISSUE THREAD

**Issue by fgladu** [broncowdd/BoZoN#170]
Bonjour,

Je reçois le message suivant en voulant uploader un fichier de 40 MB avec un nouveau compte "admin" :
**[application/x-diskcopy, 6] The file doesn't fit**
J'ai créer deux autres compte "admin", même résultat.
Le compte original n'est pas affecté par le problème.
Le max upload size a été augmenté à 4096 MB dans BoZoN et dans PHP.

Seule particularité de mon installation, **uploads -> /mnt/.../bozon/uploads/**

BoZoN 2.4 build 15
CentOS 7
PHP 5.6

Merci!

**Comment by broncowdd** [comment]
Je regarde ça dès mon retour ;-)

**Comment by joaofl** [comment]
Have similar problem. I have space on disk, but I'm getting the error: "[application/x-gzip, 495] The file doesn't fit "

It seems to be related to lack of ram memory, since I'm using an embedded arm server with few resources.

Tried to switch the temp folder location to my external hard disk (with enough space), what did not solve the issue.

Guess that, during the upload, instead of being cached on ram, files have to be written on disk, probably inside the temp folder. At least this is how Transmission torrent client does, for example.

**Comment by broncowdd** [comment]
Yes, the upload is done in the temp folder... but I don't think it's a free space problem...
I'll figure it out...

**Comment by broncowdd** [comment]
J'ai essayé avec des fichiers de plusieurs tailles sans problème...
Il faut que le fichier contienne dans l'espace **restant** du profil. Par défaut, les profils ont 50 Mo d'espace.

I've tried without any issue...
The uploaded file must fit in the remaing free space in the profile folder.
Each profile has 50 Mb of free space by default.

Have you tried to increase this value in the profiles page ?

**Comment by joaofl** [comment]
Actually did not check if it was limited by the profile. Where can I edid that?

I would say it is a bit hard to understand how to fit a user you create on one of those profiles. How to create a guest and authorize him, or any other user to fit on those other profiles.

I'll check again and let you know. Thanks!

**Comment by broncowdd** [comment]
It's easy to edit, don't worry ^^
First, clic on the «users» menu button
![capture du 2016-09-05 09 11 12](https://cloud.githubusercontent.com/assets/4249696/18240039/8b1bc71a-7349-11e6-8627-73d79b977e41.png)

Then, select the «space» tab and edit the user's value.
![capture du 2016-06-14 11 24 45](https://cloud.githubusercontent.com/assets/4249696/18240047/a03fe4f0-7349-11e6-928f-5d2ee3346920.png)

You can also edit the status (admin, guest etc), delete a profile and change the password.

I have to make a documentation ! (it's the hardest thing to do ^^)
